### DISCUSSION TITLE
A Getting Started Guide

### DISCUSSION THREAD

**Discussion by DuneArchitect** [icedream/go-stagelinq#5] [Ideas]
Hi, would it be possible to write a quick Getting Started Guide to show how to get the example application up and running to read in information from StagelinQ?

**Comment by icedream** [comment]
I will add this information as soon as I find time for this project. The procedure is relatively standard and the project was originally aimed at Go devs wanting to dive into using the protocol, and so it was skipped from documentation for a start.

For now I'll note it down here.

Make sure you have a recent enough Go version installed. The build process would be one of:

- run `go install -v github.com/icedream/go-stagelinq/cmd/...` to just let Go download the code and install the binary to `$GOPATH`
- `git clone` this repository and then run `go build -v ./cmd/stagelinq-discover` in the repository folder to build your binary into the repository folder
